portfolio/src/main/webapp/style.css

```diff
@@ -128,3 +129,63 @@ header > h1 {
 	max-width: 100%;
 	width: 1200px;
 }
+
+.container {
+	margin: 0 auto;
+	max-width: 100%;
+	width: 1200px;
```

Comment:
Some of your indentation is using tabs instead of spaces. Maybe you can use the find and replace tool to replace all tabs with the correct number of spaces?